CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ---
+## v?.?.?
+> Addition of an intern MeshService. 
```

Comment:
You can merge this with the next version header below as another Feature bullet :)

Reply:
Good to merge once that's fixed!

Reply:
done